builds/azure-pipelines/template-steps-build-test.yml

```diff
@@ -102,6 +102,15 @@ steps:
     targetFolder: $(azureFunctionsExtensionBundlePath)/bin
     overWrite: true
 
+  # Copy the Sql nupkg to ensure it's available for tests.
+- task: CopyFiles@2
+  displayName: 'Copy local Sql package to local-packages folder'
+  inputs:
+    sourceFolder: $(Build.SourcesDirectory)/src/bin/${{ parameters.configuration }}
+    contents: '*'
```

Comment:
the folder has just nupkg and snupkg, * only copies the files and not all the subfolders and it's contents.

Reply:
It would still be good to specify only *.nupkg anyways (we don't need the snupkg) - that way if something else ends up there we don't unnecessarily copy stuff and potentially cause issues.